Dockerfile

```diff
@@ -9,8 +9,8 @@ ARG NODEJS_VERSION="16"
 ARG YARN_VERSION="1.22.19"
 
 # Packages
-ARG BUILD_PACKAGES="nodejs git imagemagick build-essential libpq-dev"
-ARG RUN_PACKAGES="imagemagick shared-mime-info pkg-config libmagickcore-dev libmagickwand-dev libpq-dev libjemalloc-dev libjemalloc2"
+ARG BUILD_PACKAGES="nodejs git build-essential libpq-dev libvips42"
+ARG RUN_PACKAGES="shared-mime-info pkg-config libpq-dev libjemalloc-dev libjemalloc2 libvips42"
```

Comment:
Können wir imagemagick wirklich einfach so drop-in mässig ersetzen?
Im Gemfile.lock haben wir ja zB noch mini_magick und das scheint zumindest auf den ersten Blick von imagemagick abhängig zu sein :raised_eyebrow: 
https://rubygems.org/gems/mini_magick

Reply:
Das hochladen von Fotos scheint noch zu funktionieren, deswegen würde ich das so drin lassen.